tutorial/part1/chapter3.md

```diff
@@ -2,16 +2,16 @@
 
 ## What you will learn
 
-In this chapter we will explore various Dash **components** and how to position them within the app **layout**
+In this chapter we will explore various Dash **components** and how to position them within the app **layout**.
 
-[ending_file](./ch3_files/app.py)
+[Click here to download the complete code file for this chapter](https://github.com/open-resources/dash_curriculum/blob/main/tutorial/part1/ch2_files/chapter3_app.py)
```

Comment:
I left a comment links in chapter 2. Just wanted to make sure we're using link-formatting the way we agreed on. Whatever is best for displaying on Jupyter Books.